modules/benchmark/R/load_data.R

```diff
@@ -26,13 +27,27 @@ load_data <- function(data.path, format, start_year = NA, end_year = NA, site =
     fcn <- match.fun(fcn1)
   } else if (exists(fcn2)) {
     fcn <- match.fun(fcn2)
+  } else if (!exists(fcn1) & !exists(fcn2)) { 
+    #To Do: call to DAP to see if conversion to csv is possible
+    #Brown Dog API call through BDFiddle, requires username and password
+    #will be made into R package
+    install_github("yan130/bd.r")
+    
+    key   = get_key("https://bd-api.ncsa.illinois.edu",username,password) #generic
```

Comment:
Where, exactly, are username and password coming from here?  This is a non-trivial problem since we don't want those in the settings file, where they would get archived with the run and potentially even inserted into the database via Benchmarking.  @robkooper is there an easy way to spawn a prompt to ask for the username and password interactively? Or should there be a PEcAn default (which also seems dangerous).